spring-components/rdf4j-spring/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/Bind.java

```diff
@@ -0,0 +1,24 @@
+package org.eclipse.rdf4j.sparqlbuilder.constraint;
+
+import org.eclipse.rdf4j.sparqlbuilder.core.Assignable;
+import org.eclipse.rdf4j.sparqlbuilder.core.Variable;
+import org.eclipse.rdf4j.sparqlbuilder.graphpattern.GraphPattern;
+import org.eclipse.rdf4j.sparqlbuilder.util.SparqlBuilderUtils;
+
+public class Bind implements GraphPattern {
```

Comment:
Ah, interesting, there is actually a separate issue for that: GH-2767. This should probably be moved to the sparqlbuilder module though.

Reply:
If we're keeping this as part of this PR (which I'm fine with) we should move it to the sparqlbuilder module I think.

Reply:
I'll gladly move the code. However, there is much more in the `constraint` package in this module, including the whole `propertypath` subpackage. I suggest we move all of that to the sparqlbuilder module, keeping the package structure as is. Ok?

Reply:
I'll gladly move the code. However, there is much more in the `constraint` package in this module, including the whole `propertypath` subpackage. I suggest we move all of that to the sparqlbuilder module, keeping the package structure as is. Ok?

Reply:
Ah yes, I should have been clearer sorry, I meant this comment for the whole package, not just this particular class. Thanks!